timemachine/datasets/readme.md

```diff
@@ -3,5 +3,5 @@
 
 (Note: `freesolv.sdf` reflects a fix described in https://github.com/MobleyLab/FreeSolv/issues/48 )
 
-## `fep-benchmark`
-Datasets in the `fep-benchmark` folder are imported from v1.0 of https://github.com/MCompChem/fep-benchmark/releases/tag/v1.0 (distributed under MIT license), created by Christina Schindler and Daniel Kuhn, Merck KGaA, Darmstadt, Germany, December 2018 , accompanying https://chemrxiv.org/articles/preprint/Large-Scale_Assessment_of_Binding_Free_Energy_Calculations_in_Active_Drug_Discovery_Projects/11364884 (Schindler et al., 2019)
+## `fep_benchmark`
```

Comment:
This was caught by mypy, but is a more general issue: `fep-benchmark` is invalid as a module name. I think this would have prevented us from accessing the data using importlib.